content/applications/finance/accounting/getting_started/main_concept.rst

```diff
@@ -5,124 +5,121 @@ Main accounting concepts
+- Audit reports: general ledger, trial balance, journal report, intrastat report, check register.
+- Partner reports: partner ledger, aged receivable, aged payable.
+- Management: invoice analysis, unrealized gains/losses, depreciation schedule, disallowed expenses,
+  budget analysis, product margins, 1099 report.
+- Country specific reports based on your company's location and the fiscal localization packages
+  installed on your database.
 
-In Odoo, you can generate financial reports in real time. Odoo's
-reports range from basic accounting reports to advanced management
-reports. Odoo's reports include:
+.. tip::
+   Odoo's report engine allows you to customize your own report based on your own
+   :doc:`formulas <../reporting/overview/customize>`.
 
-* Performance reports (such as Profit and Loss, Budget Variance)
-* Position reports (such as Balance Sheet, Aged Payables, Aged
-  Receivables)
-* Cash reports (such as Bank Summary)
-* Detail reports (such as Trial Balance and General Ledger)
-* Management reports (such as Budgets, Executive Summary)
+Tax report
+----------
 
-Odoo's report engine allows you to customize your own report based on
-your own formulae.
+Odoo computes all accounting transactions for the specific tax period and uses these totals to
+calculate the tax obligation.
 
-Import bank feeds automatically
-===============================
+.. important::
+  Once the tax report has been generated, Odoo does not allow to make journal entries involving VAT
+  involving VAT in the closed period. If the period is locked, any correction to customer or vendor
+  bills have to be recorded in the next period.
 
-Bank reconciliation is a process that matches your bank statement
-lines, as supplied by the bank, to your accounting transactions in the
-general ledger. Odoo makes bank reconciliation easy by frequently
-importing bank statement lines from your bank directly into your Odoo
-account. This means you can have a daily view of your cashflow without
-having to log into your online banking or wait for your paper bank
-statements.
+.. note::
+   Depending on the country's localization, an .XML file can be generated to upload the tax report
+   directly on the VAT platform of the legal administration.
 
-Odoo speeds up bank reconciliation by matching most of your imported
-bank statement lines to your accounting transactions. Odoo also
-remembers how you've treated other bank statement lines and provides
-suggested general ledger transactions.
+Bank synchronization
+====================
 
-Calculate the tax you owe your tax authority
-============================================
+Odoo bank synchronization system directly connects with your bank institution to automatically
+import all bank transactions into your database. This means a daily view about cashflow
+will be available without having to log into the online banking system or wait for paper bank
+statements.
```

Comment:
```suggestion
Odoo bank synchronization system connects with your bank institution to automatically
import your bank transactions into your database, offering an overview of your cash flow without logging into the online banking system or waiting for paper bank statements.
```

Reply:
I find the sentence suggested a bit too long for English sentences' structure. I still kept your suggestion but made it in 2 sentences.